builder/FindMFX.cmake

```diff
@@ -35,10 +35,10 @@ endif()
 if( CMAKE_MFX_HOME )
   set( MFX_API_HOME ${CMAKE_MFX_HOME} )
 else()
-  set( MFX_API_HOME $ENV{MFX_HOME} )
+  set( MFX_API_HOME ${MFX_HOME} )
 endif()
 
-find_path( MFX_INCLUDE mfxdefs.h PATHS ${MFX_API_HOME}/include )
+find_path( MFX_INCLUDE mfxdefs.h PATHS ${MFX_API_HOME}/include NO_CMAKE_FIND_ROOT_PATH )
```

Comment:
Explain, please, why we need `NO_CMAKE_FIND_ROOT_PATH` here?

Reply:
In certain cross-compiling environments such as buildroot find_path not works properly. Correct behaivour can be  also reached by passing CMAKE_FIND_ROOT_PATH_MODE_INCLUDE parameter to cmake, but parameter affect all cases of find_file, it is not needed here.

Reply:
ok.